### DISCUSSION TITLE
How to install latest buildah release on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by sjm-ho** [containers/buildah#5657] [Q&A]
Hello community, 
I need some help with a situation I am stuck on, actually I want to install buildah on a base image of github actions arc runner which is Ubuntu 22.04 distribution

My dockerfile looks like

```
FROM ghcr.io/actions/actions-runner:2.317.0

RUN sudo apt-get update \
    && sudo apt-get install -y \
        buildah \
        awscli \
        npm \
        curl \
    && sudo rm -rf /var/lib/apt/lists/*
```
But like this it installs buildah 1.23. 
I checked the ubuntu package repository seems like it only has 1.23.1+ds1-2
https://packages.ubuntu.com/jammy/buildah

I want to install buildah release > 1.28. How can I achieve that?

**Comment by doganulus** [comment]
For GitHub Actions, you can use Ubuntu 24.04 , which has `buildah` v1.33 by default.

```
jobs:
  buildah-build:
    runs-on: ubuntu-24.04
```

**Reply by gbraad** [reply]
This is too old, as this does not do `manifest add --artifact`

**Reply by mmm8955405** [reply]
Irresponsible, directly releasing the source code and forcing programmers to compile from scratch, even Ubuntu 24.04, 1.33 is too old, and 1.39 has fixed many bugs

**Reply by rhatdan** [reply]
Irresponsible of who? Upstream is not responsible for releasing to any or every distribution or older version.  That is the job of volunteers or people paid to do it for particular distributions.  If Canonical wants to package software for its distribution, it can pay people to update versions. Otherwise we rely on volunteers to package software for these distributions. If you would like to step forward to maintain software for Ubuntu, we welcome you.
